src/components/UserDetails/index.jsx

```diff
@@ -10,13 +10,15 @@ export default function UserDetails({ header = false }) {
     const userCohort = currentUser?.cohortId ? `, Cohort ${currentUser.cohortId}` : ''
 
     const userFullName = `${currentUser?.firstName} ${currentUser?.lastName}`
+    
+    const stringToTitleCase = string => string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
 
     if (header) {        
         return (
             <div className="post-user-name">
                 <p>{userFullName}</p>
                 <small>
-                    {currentUser.role} {userCohort}
+                    {stringToTitleCase(currentUser.role)}{userCohort}
```

Comment:
Should there be a space before injecting the userCohort?

Reply:
![image](https://github.com/user-attachments/assets/b03c53a6-b5ff-44ff-99d9-5542bd33f0bd)
It renders correctly